app/src/main/java/com/naman14/timberx/ui/dialogs/AddToPlaylistDialog.kt

```diff
@@ -60,24 +79,4 @@ class AddToPlaylistDialog : DialogFragment() {
             }
         }
     }
-
-    companion object {
-        @JvmOverloads
```

Comment:
FYI, you'd only use this for interop with regular Java, but it has no purpose with Kotlin other than increasing method count 😄

Reply:
Yeah, makes sense